packages/strapi-admin/admin/src/translations/pt.json

```diff
@@ -1,173 +1,217 @@
   "app.components.Button.save": "Guardar",
+  "app.components.Button.reset": "Restaurar",
   "app.components.ComingSoonPage.comingSoon": "Em breve",
-  "app.components.ComingSoonPage.featuresNotAvailable": "Esta funcionalidade continua em desenvolvimento",
-  "app.components.DownloadInfo.download": "Transferência em curso...",
+  "app.components.ComingSoonPage.featuresNotAvailable": "Esta funcionalidade ainda se encontra em desenvolvimento",
+  "app.components.DownloadInfo.download": "Transferência em progresso...",
   "app.components.DownloadInfo.text": "Isto poderá levar alguns minutos. Obrigado pela sua paciência",
-  "app.components.EmptyAttributes.title": "Ainda não há campos",
-  "app.components.HomePage.button.blog": "VEJA MAIS NO BLOG",
-  "app.components.HomePage.button.quickStart": "COMEÇAR UM BREVE TUTORIAL",
+  "app.components.EmptyAttributes.title": "Ainda não há atributos",
+  "app.components.HomePage.button.blog": "VÊ MAIS NO BLOG",
+  "app.components.HomePage.button.quickStart": "COMEÇAR O GUIA DE INICIAÇÃO RÁPIDA",
   "app.components.HomePage.community": "Encontre a comunidade na web",
-  "app.components.HomePage.community.content": "Converse com membros da equipe, colaboradores e desenvolvedores em diferentes canais.",
+  "app.components.HomePage.community.content": "Conversa com membros da equipa, contribuidores e desenvolvedores através de diferentes canais.",
+  "app.components.HomePage.create": "Cria o teu primeiro Tipo de Conteúdo",
   "app.components.HomePage.createBlock.content.first": "A ",
-  "app.components.HomePage.createBlock.content.second": " extensão irá ajudar-vos a definir a estrutura de dados de seus modelos. Se é novo aqui, nós recomendamos fortemente que você siga o nosso ",
+  "app.components.HomePage.createBlock.content.second": " extensão irá ajudar-te a definir a estrutura de dados dos teus modelos. Se és novo aqui, recomendamos fortemente que sigas o nosso ",
   "app.components.HomePage.createBlock.content.tutorial": " tutorial.",
-  "app.components.HomePage.cta": "CONFIRME",
-  "app.components.HomePage.newsLetter": "Subscreva-se à newsletter para ficar a par sobre Strapi",
+  "app.components.HomePage.cta": "CONFIRMAR",
+  "app.components.HomePage.newsLetter": "Subscreve-te à newsletter para ficares a par do Strapi",
   "app.components.HomePage.support": "SUPORTE-NOS",
-  "app.components.HomePage.support.content": "Comprando a camisola, permitirá-nos continuar o nosso trabalho no projecto para dar à si a melhor experiência possível!",
-  "app.components.HomePage.support.link": "OBTENHA A SUA CAMISOLA AGORA",
+  "app.components.HomePage.support.content": "Ao comprar a t-shirt, irá permitir-nos continuar o nosso trabalho no projeto para te proporcionar a melhor experiência possível!",
+  "app.components.HomePage.support.link": "OBTENHA A SUA T-SHIRT AGORA",
   "app.components.HomePage.welcome": "Bem-vindo(a) a bordo",
   "app.components.HomePage.welcome.again": "Bem-vindo(a) ",
```

Comment:
These "Bem-vindo" are in masculine/feminine, while [here](https://github.com/strapi/strapi/blob/9ed11a66829dbdafabb49cb8813dd4d891a247de/packages/strapi-admin/admin/src/translations/pt.json#L219) it's the default (masculine).

I'll analyse the file better open an issue if no one does but unfortunately I don't have much time now.

Reply:
I'd go for m/f all the time.